src/fal/telemetry/telemetry.py

```diff
@@ -330,3 +333,24 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return _log_call
+
+
+def _remove_path_from_props(props: Dict) -> Dict:
+    PATH_STR = "[PATH]"
+
+    def _replace_value(val):
+        if isinstance(val, str):
+            if os.path.exists(val):
+                val = PATH_STR
```

Comment:
What about keeping the basename?
```
>>> x
'/Users/matteo/Projects/fal/fal/integration_tests'
>>> '[PATH]/' + os.path.basename(x)
'[PATH]/integration_tests'
```

Reply:
I'd rather not, since we don't know what the basename might contain. An internal dbt project name might be a secret for example.